js/main.js

```diff
@@ -19,3 +19,69 @@ const getRandomFloatNumber = (min, max, afterPoint) => {
 };
 
 getRandomFloatNumber(3, 6, 2);
+
+
+//ДЗ по 3 лекции
+
+import _ from 'lodash';
+const typePremises = ['palace', 'flat', 'house', 'bungalow'];
+const timeCheckinCheckout = ['12:00', '13:00', '14:00'];
+const facilities = ['wifi', 'dishwasher', 'parking', 'washer', 'elevator', 'conditioner'];
+const photoGallery = ['http://o0.github.io/assets/images/tokyo/hotel1.jpg', 'http://o0.github.io/assets/images/tokyo/hotel2.jpg', 'http://o0.github.io/assets/images/tokyo/hotel3.jpg'];
+
+const createAuthor = () => {
+  const imageName = 'img/avatars/user' + '0' + getRandomNumber(1, 8) + '.png';
+  return {
+    avatar: imageName,
+  };
```

Comment:
Переменная imageName не нужна, и лучше записать так
```suggestion
  return {
    avatar: `img/avatars/user0${getRandomNumber(1, 8)}.png`,
  };
```